tool/gh_actions/update_directory.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# set_up.sh
+# GitHub Actions step: Set up the files directory
+#
+# 2023 May 3
+# Author: Yao Jing Quek <yao.jing.quek@intel.com>
+
+set -euo pipefail
+
+root_directory=$(git rev-parse --show-toplevel)
+
+src_path="$root_directory/src"
+rohd_submodule_path="$root_directory/rohd"
+
+git submodule update --init --recursive
+git submodule update --remote rohd
+
+for directory in "$rohd_submodule_path/doc/website"/*; do
```

Comment:
I think it's a little weird to call the path `doc/website` in the `rohd` repo since it's not really a website in that context.  Can we organize the directory structure in a way that makes sense for general documentation instead of website-specific in the `rohd` repo?

Also, is the naming of the files something related to jekyl with `A01`, `A02`, etc. or is it just for sorting?

Reply:
It's just for sorting.

Reply:
Ok then, seems fine to me for the file naming, but the comment on `website` folder stands

Reply:
docs/user_guide